src/python/google_cloud_utils/big_query.py

```diff
@@ -379,8 +379,18 @@ def insert(self, inserts):
       time.sleep(1)
 
       if not result:
+        # Use result from the first batch, appending errors from the rest.
         result = response
       else:
-        result['insertErrors'].extend(response['insertErrors'])
+        # If there are new errors from the current batch, append to the result.
```

Comment:
I feel the else makes the flow of the logic a bit confusing here. We can probably just continue after line 383 and unindent the rest.

Reply:
thanks, done!